helix-core/src/main/java/org/apache/helix/model/CloudConfig.java

```diff
@@ -94,6 +94,10 @@ public CloudConfig(String cluster, boolean enabled, CloudProvider cloudProvider,
     }
   }
 
+  public static CloudConfig buildEmptyCloudConfig() {
```

Comment:
Do you think is it better to change this to a default constructor?

Reply:
Good point. Updated.